changelog.md

```diff
@@ -4,6 +4,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v6.0.1...HEAD)
+
+### Fixed
+* [PCM-1693](https://inindca.atlassian.net/browse/PCM-1693) - Fixed `client.ts` file by adding in check for `audioTracks` size as well as passing in track in question that one is trying to remove.
```

Comment:
Can we reword this please? It's a bit hard to read and this is customer facing.

Reply:
Absolutely. I'm sorry, I didn't realize this was customer facing.